### DISCUSSION TITLE
Show machine type in status bar in VSCode

### DISCUSSION THREAD

**Discussion by SmallhillCZ** [community/community#60990] [Codespaces]
### Select Topic Area

Product Feedback

### Body

When developing in the codespace, show mahcine type (e.g. `CPU: 4, RAM: 8GB`) in the VSCode's status bar.

I sometimes increase my codespace machine type for some task, but then I forget to take it down and it is not clearly visible while developing.

**Comment by eljog** [comment]
Machine details can be seen by running the command `Codespaces: Detail` in vscode.

<img width="334" alt="image" src="https://github.com/community/community/assets/15967687/a471d8b3-95f4-4ca1-9d17-b54cf4ee760c">

**Reply by SmallhillCZ** [reply]
Yes, I know, but for this you already need to know you have to check so doesn't solve the case when you accidentlally leave your machine on 32 CPUa and pay eight times more than you wanted, which is what I have explained in the original post. Having an option to show the basic CPU+RAM information in your eyesight would.
